modules/services/mako.nix

```
@@ -339,5 +359,26 @@ in {
         ${cfg.extraConfig}
       '';
     };
+
+    systemd.user.services.mako = {
+      Unit = {
+        Description = "Lightweight Wayland notifcation deamon";
+        Documentation = "man:mako(1)";
+        PartOf = [ "graphical-session.target" ];
+        After = [ "graphical-session-pre.target" ];
+      };
+
+      Service = {
+        Type = "dbus";
+        BusName = "org.freedesktop.Notifications";
+        ExecCondition = "/bin/sh -c '[ -n \"$WAYLAND_DISPLAY\" ]'";
+        ExecStart = getExe cfg.package;
+        ExecReload = "${cfg.package}/bin/makoctl reload";
+      };
+
+    } // (if autoStart then {
+      Install.WantedBy = [ "graphical-session.target" ];
+    } else
+      { });
```

Is this the way to do something like this?

---

You probably want `lib.optional` instead.

```suggestion
      Install.WantedBy = lib.optional autoStart "graphical-session.target";
    };
```